pkgs/applications/virtualization/pid1/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage {
+  pname = "pid1";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "fpco";
+    repo = "pid1-rs";
+    # Contains fixes for Cargo.lock. Should be updated to 0.1.2 when
+    # that gets released.
+    rev = "be6f061";
```

Comment:
Can you tag a new release so this can be `"v${version}"`?

Reply:
Fixed, thanks!